mta-otp-proxy/src/main/java/com/camsys/shims/gtfsrt/alerts/siri/deserializer/SiriDeserializerWithModifications.java

```diff
@@ -78,4 +79,27 @@ private void removeHtml(Siri siri) {
             }
         }
     }
+
+    private void filterLmm(Siri siri) {
+        //ServiceDelivery serviceDelivery = new ServiceDelivery();
+        //siri1.setServiceDelivery(serviceDelivery);
+        //siri1.setServiceDelivery(siri.getServiceDelivery());
+        Situations s1 = new Situations();
+        if (siri.getServiceDelivery() != null) {
+            ServiceDelivery sd = siri.getServiceDelivery();
+            if (sd.getSituationExchangeDelivery() != null) {
+                for (SituationExchangeDeliveryStructure seds : sd.getSituationExchangeDelivery()) {
+                    if (seds.getSituations() != null) {
+                        Situations s = seds.getSituations();
+                        if (s.getPtSituationElement() != null) {
+                            for (PtSituationElementStructure pt: s.getPtSituationElement()) {
+                                if (!pt.getSituationNumber().getValue().contains("lmm")) s1.getPtSituationElement().add(pt);
```

Comment:
I would prefer to make this more specific.  Make it startsWith("lmm:").  The GMS situation numbers are quite large and I could see lmm appearing and causing some very strange bugs

Reply:
Switched to `startsWith` from `contains`